iloop_to_model/__init__.py

```diff
@@ -5,11 +5,23 @@
 import requests
 from potion_client import Client
 from potion_client.auth import HTTPBearerAuth
+from raven import Client as RavenClient
+from raven.conf import setup_logging
+from raven.handlers.logging import SentryHandler
+
+from . import settings
+
 
 logger = logging.getLogger('iloop-to-model')
 logger.addHandler(logging.StreamHandler(stream=sys.stdout))  # Logspout captures logs from stdout if docker containers
 logger.setLevel(logging.DEBUG)
 
+# Configure Raven to capture warning logs
+raven_client = RavenClient(settings.Default.SENTRY_DSN)
```

Comment:
What happens if the DSN is not present -> raven and Sentry shouldn't be used (eg local development)

Reply:
Yes, in the settings this value defaults to empty string which is accepted and silences the Raven client